### PR TITLE
Removed fastmem from Android UI

### PR DESCRIPTION
**Pull request by aserna3**
@Sonicadvance1 recommended this.

### CODE REVIEW

**Comment by degasus** [comment]
I doubt this is enough. Can you search for "fastmemPref" in the Android directory?

**Comment by aserna3** [comment]
Oh, do I need to remove the other references to fastmem in the Userpreferences.java file? I still want it to pick up whatever the user changes in the ini and my shield is refusing to write dolphin's config files to storage for some stupid reason so I literally can't test this.

**Comment by degasus** [comment]
Reading the ini is done in core, so you can remove fastmem everywhere android specific.

File: Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/utils/UserPreferences.java [hunk 1]
```diff
@@ -133,8 +132,6 @@ public static void SavePrefsToIni(Context ctx)
 	{
 		SharedPreferences prefs = PreferenceManager.getDefaultSharedPreferences(ctx);
 
-		// Whether or not the user is using dual core.
-		boolean isUsingDualCore = prefs.getBoolean("dualCorePref", true);
```
Comment (degasus) [inline Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/utils/UserPreferences.java]: of by 6 lines?

**Comment by waddlesplash** [comment]
LGTM

**Pull request merged**